src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,493 @@
+            "failure_phase": "Join",
+            "ip_address": "10.10.124.162",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.f920",
+        },
+        "706d.053a.f920": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.150.239",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.f924",
+        },
+        "706d.053a.f9c0": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.198.166",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.f938",
+        },
+        "706d.053a.fa40": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.108.120",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.f948",
+        },
+        "706d.053a.fcc0": {
+            "ap_name": "visitors-c9105",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.40.223",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.f998",
+        },
+        "706d.053a.fda0": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.71.91",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.f9b4",
+        },
+        "706d.053b.0180": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.100.242",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.fa30",
+        },
+        "706d.053b.04e0": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.23.46",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.fa9c",
+        },
+        "706d.053b.0720": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.105.129",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.fae4",
+        },
+        "706d.053b.0b00": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.194.10",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.fb60",
+        },
+        "706d.053b.0d20": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Max Retransmission to " "AP",
```

Comment:
I'll look into why the parser is adding `" "`

Reply:
I was using `pprint()` during testing and it added the extra `" "`. Fixed now! Waiting on review from @tahigash